### ISSUE TITLE
Do we want to recenter with center_of_mass, or rather geometric_center?

### ISSUE THREAD

**Issue by adrianusler** [Molara-Lab/Molara#516]
We calculate both already in the Structure class. We use center_of_mass right now, but I believe geometric_center would make more sense.

**Comment by ab5424** [comment]
This should rarely make a significant difference, I guess?

**Comment by adrianusler** [comment]
> This should rarely make a significant difference, I guess?

Agreed. I think it makes more sense, though, and there might be differences in some cases. If we can all agree that it would be ok to change it to the geometric center, then I would just do that. @ab5424 @Michel-Heinz

**Comment by Michel-Heinz** [comment]
The geometric center would be best, since the camera should look at a place where it sees most of the molecule. But we should not translate the molecule, but rather the camera to the molecule. That means the whole routine need to be reworked. This might become important fro some dipole or multipole moments
